contracts/services/osmosis-gamm-lper/src/msg.rs

```diff
@@ -0,0 +1,177 @@
+use std::str::FromStr;
+
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{
+    ensure, Addr, Decimal, Deps, DepsMut, Empty, StdError, StdResult, Uint128, Uint64,
+};
+use cw_ownable::cw_ownable_query;
+use osmosis_std::types::osmosis::{gamm::v1beta1::Pool, poolmanager::v1beta1::PoolmanagerQuerier};
+
+use valence_macros::OptionalStruct;
+use valence_osmosis_utils::utils::DecimalRange;
+use valence_service_utils::{
+    error::ServiceError, msg::ServiceConfigValidation, ServiceAccountType, ServiceConfigInterface,
+};
+#[cw_serde]
+pub enum ActionsMsgs {
```

Comment:
forgot this, In my other PR I renamed all to ActionMsgs (some had ActionsMsgs and others ActionMsgs). Can we rename it here too to have it consistent everywhere